app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,31 @@
+class LikesController < ApplicationController
+  def create
+    @bookmark = Bookmark.find(params[:bookmark_id])
+    @topic = Topic.find(params[:topic_id])
+    like = current_user.likes.build(bookmark: @bookmark)
```

Comment:
Nice work using a local variable for `like` when an instance variables isn't' needed.